db/seeds.rb

```diff
@@ -6,6 +6,8 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+AdminUser.create!(email: 'admin_features@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
Why do you want to create that user in development only?

Reply:
It was autogenerated when I setup active admin initially.  Will change it.